ast/src/main/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitor.java

```diff
@@ -570,8 +570,36 @@ public void visitTernaryExpr(TernaryExpr ternaryExpr) {
     visit(ternaryExpr.getElseExpr());
   }
 
+  private boolean maybeEmitKnownUniformMacro(ArrayIndexExpr arrayIndexExpr,
+                                           String knownUniformArrayName, String prefix) {
+    /* If arrayIndexExpr indexes into a literal-to-uniform array, output a macro name instead. */
```

Comment:
FYI: We prefer line comments (//) in this project, unless there's a good reason not to use them (no action required as it's not mandatory).